<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P76104833-hw1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>4. PCA</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn4_1">
         <property name="text">
          <string>4.1 Image Reconstruction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn4_2">
         <property name="text">
          <string>4.2 Compute the Reconstruction Error</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>5. ResNet50</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btn5_1">
         <property name="text">
          <string>1. Show Model Structures</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn5_2">
         <property name="text">
          <string>2. Show TenSorBoard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn5_3">
         <property name="text">
          <string>3. Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn5_4">
         <property name="text">
          <string>4. Random-Erasing</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
